commit c2a354ce545847debaf1942adaddc31b0ec3a4b8
Author: Lucas De Marchi <lucas.demarchi@intel.com>
Date:   Fri Jan 25 14:24:42 2019 -0800

    drm/i915: always return something on DDI clock selection
    
    commit 2a121030d4ee3f84f60c6f415f9c44bffbcde81d upstream.
    
    Even if we don't have the correct clock and get a warning, we should not
    skip the return.
    
    v2: improve commit message (from Joonas)
    
    Fixes: 1fa11ee2d9d0 ("drm/i915/icl: start adding the TBT pll")
    Cc: Paulo Zanoni <paulo.r.zanoni@intel.com>
    Cc: <stable@vger.kernel.org> # v4.19+
    Signed-off-by: Lucas De Marchi <lucas.demarchi@intel.com>
    Reviewed-by: Mika Kahola <mika.kahola@intel.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/20190125222444.19926-3-lucas.demarchi@intel.com
    (cherry picked from commit 7a61a6dec3dfb9f2e8c39a337580a3c3036c5cdf)
    Signed-off-by: Jani Nikula <jani.nikula@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/i915/intel_ddi.c b/drivers/gpu/drm/i915/intel_ddi.c
index c9af34861d9e..b4b1f9ca05b6 100644
--- a/drivers/gpu/drm/i915/intel_ddi.c
+++ b/drivers/gpu/drm/i915/intel_ddi.c
@@ -1085,7 +1085,7 @@ static uint32_t icl_pll_to_ddi_pll_sel(struct intel_encoder *encoder,
 			return DDI_CLK_SEL_TBT_810;
 		default:
 			MISSING_CASE(clock);
-			break;
+			return DDI_CLK_SEL_NONE;
 		}
 	case DPLL_ID_ICL_MGPLL1:
 	case DPLL_ID_ICL_MGPLL2:
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1340_c2a354ce545847debaf1942adaddc31b0ec3a4b8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1340_c2a354ce545847debaf1942adaddc31b0ec3a4b8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
